hoomd/md/update.py

```diff
@@ -221,100 +224,144 @@ class mueller_plathe_flow(_updater):
     the "max" and "min" slap might be swapped.
 
     Args:
-        group (``hoomd.group``): Group for which the update will be set
-        flow_target (:py:mod:`hoomd.variant`): Integrated target flow. The unit is the in the natural units of the simulation: [flow_target] = [timesteps] x :math:`\mathcal{M}` x :math:`\frac{\mathcal{D}}{\tau}`. The unit of [timesteps] is your discretization dt x :math:`\mathcal{\tau}`.
-        slab_direction (``X``, ``Y``, or ``Z``): Direction perpendicular to the slabs..
-        flow_direction (``X``, ``Y``, or ``Z``): Direction of the flow..
-        n_slabs (int): Number of slabs. You want as many as possible for small disturbed volume, where the unphysical swapping is done. But each slab has to contain a sufficient number of particle.
-        max_slab (int): Id < n_slabs where the max velocity component is search for. If set < 0 the value is set to its default n_slabs/2.
-        min_slab (int): Id < n_slabs where the min velocity component is search for. If set < 0 the value is set to its default 0.
+        filter (`hoomd.filter.ParticleFilter`): Subset of particles on which to
+            apply this updater.
+
+        flow_target (`hoomd.variant.Variant`): Integrated target flow. The unit
+            is the in the natural units of the simulation: [flow_target] = [timesteps] x :math:`\mathcal{M}` x :math:`\frac{\mathcal{D}}{\tau}`.
+            The unit of [timesteps] is your discretization dt x :math:`\mathcal{\tau}`.
+
+        slab_direction (``X``, ``Y``, or ``Z``): Direction perpendicular to the slabs.
+
+        flow_direction (``X``, ``Y``, or ``Z``): Direction of the flow.
+
+        n_slabs (int): Number of slabs. You want as many as possible for small
+            disturbed volume, where the unphysical swapping is done. But each
+            slab has to contain a sufficient number of particle.
```

Comment:
What does "sufficient" mean here?

Reply:
@mphoward Could you recommend some language here?

Reply:
Sufficient = "big enough volume that there are enough particles to apply momentum swap to". So, it depends on particle density, volume of the slab, and how much momentum to swap.

Not sure if this belongs here or maybe in the "long description", but some language along these lines should be in the docs somewhere:
```
n_slabs (int): Number of slabs used to divide the simulation box along the shear gradient. Using too few slabs will lead to a larger volume being disturbed by the momentum exchange, while using too many slabs may mean that there are not enough particles to exchange the target momentum.
```